_tools/process-data.ts

```diff
@@ -93,19 +94,31 @@ for (let page of targetData) {
 
     // Then make a list of only the proxies that are live
     const proxyList: ResultProxy[] = [];
-    for (let proxyKey of Object.keys(contents.proxies)) {
-      const proxy = contents.proxies[proxyKey];
-      if (liveContracts[proxy.aggregator] && !proxy.name.includes("Healthcheck")) {
+    if (contents.proxies) {
+      for (let proxyKey of Object.keys(contents.proxies)) {
+        const proxy = contents.proxies[proxyKey];
+        if (liveContracts[proxy.aggregator] && !proxy.name.includes("Healthcheck")) {
+          proxyList.push({
+            pair: proxy.name,
+            deviationThreshold: liveContracts[proxy.aggregator].deviationThreshold,
+            heartbeat: liveContracts[proxy.aggregator].heartbeat,
+            decimals: liveContracts[proxy.aggregator].decimals,
+            proxy: proxyKey.startsWith("sol-") ? proxyKey.replace("sol-", "") : proxyKey,
+          });
+        }
+      }
+    } else {
+      for (let contractKey of Object.keys(contents.contracts)) {
+        const contract = contents.contracts[contractKey];
         proxyList.push({
-          pair: proxy.name,
-          deviationThreshold: liveContracts[proxy.aggregator].deviationThreshold,
-          heartbeat: liveContracts[proxy.aggregator].heartbeat,
-          decimals: liveContracts[proxy.aggregator].decimals,
-          proxy: proxyKey,
+          pair: contract.name,
+          deviationThreshold: liveContracts[contractKey].deviationThreshold,
```

Comment:
```suggestion
            deviationThreshold: 'N/A',
```
This ends up being `null`. Can we override it with "N/A"?

Reply:
`deviationThreshold` is a number so I tried to replace it with NaN but it returns null in any case. In the table, it shows up as N/A which is fine.